### PR TITLE
Fixes #12907 - Truncates too long descriptions

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: app/models/job_invocation.rb [hunk 1]
```diff
@@ -40,6 +40,8 @@ class JobInvocation < ActiveRecord::Base
 
   default_scope -> { order('job_invocations.id DESC') }
 
+  validates :description_format, :length => { :maximum => 255 }, :if => :present?
+
```
Comment (ohadlevy) [inline app/models/job_invocation.rb]: why? please use validates_lengths_from_database

Comment (adamruzicka) [inline app/models/job_invocation.rb]: I didn't know a validator like that existed, changed to validating/truncating based on the column's length

File: test/unit/job_invocation_test.rb [hunk 2]
```diff
@@ -61,6 +61,16 @@
...
       end
+
+      it 'truncates generated description to 255 characters' do
+        limit = 255
+        JobInvocation.columns_hash['description'].expects(:limit).returns(limit)
+        job_invocation.expects(:save!)
+        job_invocation.description_format = '%{job_name}'
+        job_invocation.job_name = 'a' * 1000
+        job_invocation.generate_description!
+        job_invocation.description.must_equal 'a'*limit
```
Comment (ares) [inline test/unit/job_invocation_test.rb]: `'a' * limit` as few lines above, also maybe `job_invocation.description.lenght.must_equal limit` would be more descriptive

**Comment by ares** [comment]
Except one nit good to merge.

**Comment by ares** [comment]
APJ

**Comment by ares** [comment]
Thanks @adamruzicka, merging!

**Pull request merged**